util/src/bin/dns.rs

```diff
@@ -218,6 +222,13 @@ struct DeleteRecordOpt {
     rdata: Vec<String>,
 }
 
+/// Fetch the dnskeys (key-signing-keys) from a zone, if zone is not specified it defaults to the Root, `.`
+#[derive(Debug, Args)]
+struct FetchKeysOpt {
+    /// If specified, files of Keys not in the Hickory TrustAnchor will be written to this path
+    output_path: Option<PathBuf>,
```

Comment:
Nit: `output_path` seems unnecessarily verbose for this, just `output` should do? (Especially in conjunction with the field's type.)

Reply:
This name will appear in command line help, so I think being a little more verbose here is fine

Reply:
I am aware, and it's exactly why I think being more concise would be better. `curl` makes do with `--output`, I really don't think `--output-path` is clearer.

Reply:
This one's a positional argument, so no flag is needed

Reply:
I don't have a strong opinion on this. For reference, here's the help output:

```
Usage: dns --nameserver <NAMESERVER> fetch-keys [OUTPUT_PATH]

Arguments:
  [OUTPUT_PATH]  If specified, files of Keys not in the Hickory TrustAnchor will be written to this path
```

Reply:
I will say, one reason I like path here is that this is a directory (`output_dir` would be just as good). Output I feel like is generally used when it's a single file being written to as output.

Reply:
In fact, I like output_dir better for that reason, I'm going to change to that. Let me know if you still have concerns @djc